addon_service/migrations/0001_initial.py

```diff
@@ -381,6 +382,7 @@ class Migration(migrations.Migration):
                 null=True,
                 on_delete=django.db.models.deletion.CASCADE,
                 related_name="authorized_storage_accounts",
+                related_query_name="%(class)s_authorized_storage_account",
```

Comment:
need to re-make migrations?

Reply:
ORM says that no changes were detected

Reply:
Then what's the source of this change?

Reply:
(this is the one lingering thing I'm curious about before I merge)